src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -35,51 +57,179 @@ void getLines() {
         // given
         // 지하철_노선_등록되어_있음
         // 지하철_노선_등록되어_있음
+        ExtractableResponse<Response> firstLine = createLine("1호선", "navy");
+        ExtractableResponse<Response> secondLine = createLine("2호선", "green");
 
         // when
         // 지하철_노선_목록_조회_요청
+        ExtractableResponse<Response> response = getLine("");
 
         // then
         // 지하철_노선_목록_응답됨
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
         // 지하철_노선_목록_포함됨
+        List<Long> expectIds = Arrays.asList(firstLine, secondLine).stream()
+            .map(this::getIdWithResponse)
+            .collect(Collectors.toList());
+        List<Long> resultIds = getIdsWithResponse(response);
+        assertThat(checkIdsEquals(expectIds, resultIds)).isTrue();
     }
 
     @DisplayName("지하철 노선을 조회한다.")
     @Test
     void getLine() {
         // given
         // 지하철_노선_등록되어_있음
+        createLine("2호선", "green");
 
         // when
         // 지하철_노선_조회_요청
+        ExtractableResponse<Response> response = getLine("/1");
 
         // then
         // 지하철_노선_응답됨
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
```

Comment:
```suggestion
        // given
        String expected = "2호선";
        Long id = mapId(createLine(expected, "green"));

        // when
        ExtractableResponse<Response> actual = getLine("/" + id);

        // then
        assertThat(actual.statusCode()).isEqualTo(HttpStatus.OK.value());
        assertThat(mapResponse(actual).getName()).isEqualTo(expected);
```

응답 값으로 조회하여 테스트하는 건 어떨까요
값을 하드코딩하면, 반복적인 테스트를 진행할 때 영향을 받을 수 있습니다.

Reply:
해당부분 확인하여 수정사항 반영되도록 하였습니다!